pep_sphinx_extensions/pep_zero_generator/author.py

```diff
@@ -35,7 +35,12 @@ def parse_author_email(author_email_tuple: tuple[str, str], authors_overrides: d
 
     if name_parts.surname[1] == ".":
         # Add an escape to avoid docutils turning `v.` into `22.`.
-        name_parts.surname = f"\\{name_parts.surname}"
+        name_parts = _Name(
```

Comment:
maybe use `._replace`? Though if this is currently broken, and Martin von Löwis' name renders correctly currently (it does), perhaps this branch should just be removed?

Reply:
Yes, his name is in `AUTHOR_OVERRIDES.csv` so we short circuit a few lines before this bit. Will remove this branch.